composer.json

```diff
@@ -2,6 +2,7 @@
   "name": "bitbull/magento2-module-instagramwidget",
   "description": "Instagram stream for Magento2",
   "license": "proprietary",
+  "version": "1.0.1",
```

Comment:
Why did you choose to put it here? So we must always keep it aligned with our composer.json.  There is a reason for doing so?